phys/module_physics_init.F

```diff
@@ -3331,19 +3387,23 @@ SUBROUTINE bl_init(STEPBL,BLDT,DT,RUBLTEN,RVBLTEN,RTHBLTEN,  &
                               A_E_BEP,B_U_BEP,B_V_BEP,                         & !multi-layer urban
                               B_T_BEP,B_Q_BEP,B_E_BEP,DLG_BEP,                 & !multi-layer urban
                               DL_U_BEP,SF_BEP,VL_BEP,                          & !multi-layer urban
-                              FRC_URB2D, UTYPE_URB2D,config_flags%use_wudapt_lcz)  !urban
-                             max_utype_urb2d = maxval(UTYPE_URB2D)*1.0
+                              FRC_URB2D, UTYPE_URB2D,config_flags%use_wudapt_lcz)!urban
+
+               max_utype_urb2d = maxval(UTYPE_URB2D)*1.0
 #if ( defined( DM_PARALLEL ) && ( ! defined( STUBMPI ) ) )
                max_utype_urb2d = wrf_dm_max_real(max_utype_urb2d)
 #endif
-               IF (config_flags%use_wudapt_lcz.eq.0 .and. max_utype_urb2d.gt.3.0) THEN  !new LCZ
+
```

Comment:
wudapt related - not doing anything different

Reply:
This is just adding a blank line to separate the assignment of the variable and an IF test for an error condition. Added for readability.